docs/framework/data/adonet/sql/linq/frequently-asked-questions.yml

```diff
@@ -0,0 +1,253 @@
+### YamlMime:FAQ
+metadata:
+  description: "Learn more about: Frequently Asked Questions"
+  title: "Frequently Asked Questions"
+  ms.date: "03/30/2017"
+  dev_langs:
+    - "csharp"
+    - "vb"
+  ms.assetid: 252ed666-0679-4eea-b71b-2f14117ef443
+    
+title: Frequently Asked Questions
+summary: |
+  The following sections answer some common issues that you might encounter when you implement LINQ.
+  
+  Additional issues are addressed in [Troubleshooting](troubleshooting.md).
+  
+
+sections:
+  - name: Cannot Connect
+    questions:
+      - question: |
+          I cannot connect to my database.
+        answer: |
+          Make sure your connection string is correct and that your SQL Server instance is running. Note also that [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)] requires the Named Pipes protocol to be enabled. For more information, see [Learning by Walkthroughs](learning-by-walkthroughs.md).
+          
+  - name: Changes to Database Lost
+    questions:
+      - question: |
+          I made a change to data in the database, but when I reran my application, the change was no longer there.
+        answer: |
+          Make sure that you call <xref:System.Data.Linq.DataContext.SubmitChanges%2A> to save results to the database.
+          
+  - name: "Database Connection: Open How Long?"
+    questions:
+      - question: |
+          How long does my database connection remain open?
+        answer: |
+          A connection typically remains open until you consume the query results. If you expect to take time to process all the results and are not opposed to caching the results, apply <xref:System.Linq.Enumerable.ToList%2A> to the query. In common scenarios where each object is processed only one time, the streaming model is superior in both `DataReader` and [!INCLUDE[vbtecdlinq](../../../../../../includes/vbtecdlinq-md.md)].
```

Comment:
This `[!INCLUDE]` is not rendering correctly, I do not believe the that YML supports the include syntax like MD files do.

Reply:
Under Question [!INCLUDE] doesn't support. We can use it in Answer or summary.